java/riege.importorder

```diff
@@ -0,0 +1,21 @@
+#Organize Import Order
+#Thu Jul 20 16:55:53 CEST 2023
+11=\#java
```

Comment:
Shouldn't `io` be the first one here as well?

Reply:
I will double-check with the Checkstyle configuration.